prow/plugins/verify-owners/verify-owners_test.go

```diff
@@ -292,3 +294,134 @@ func TestHandle(t *testing.T) {
 		}
 	}
 }
+
+func TestParseOwnersFile(t *testing.T) {
```

Comment:
Shouldn't we also make this test the actual parsed output?

Reply:
You mean the message? If so please encapsulate all the message in either consts or functions. Not entirely convinced its necessary

Reply:
@stevekuznetsov your call...

Reply:
No no, what Erick said is right